thunderfx failure with LitGPT ci

We are seeing errors in the litgpt CI from one of the ThunderFX tests.
e.g. https://dev.azure.com/Lightning-AI/lit%20Models/_build/results?buildId=232652&view=logs&j=47e66f3c-897a-5428-da11-bf5c7745762e&t=8ec4ada6-1a54-5cba-5628-67d3de5b9e16

There seem unrelated to changes in litgpt, so likely something in ThunderFX and / or PyTorch.
  
```
           raise TypeError(f"tree_flatten of type {type(args)} is not supported.")
E           torch._dynamo.exc.BackendCompilerFailed: backend='<thunder.dynamo.compiler.ThunderCompiler object at 0x7fd84ecee6b0>' raised:
E           TypeError: tree_flatten of type <class 'torch.SymInt'> is not supported.
E           
E           Set TORCH_LOGS="+dynamo" and TORCHDYNAMO_VERBOSE=1 for more information
E           
E           
E           You can suppress this exception and fall back to eager by setting:
E               import torch._dynamo
E               torch._dynamo.config.suppress_errors = True

/usr/local/lib/python3.10/dist-packages/thunder/core/pytree.py:78: BackendCompilerFailed
----------------------------- Captured stderr call -----------------------------
Sliding Window Attention is enabled but not implemented for `sdpa`; unexpected results may be encountered.
------------------------------ Captured log call -------------------------------
WARNING  transformers.models.qwen2.modeling_qwen2:logging.py:329 Sliding Window Attention is enabled but not implemented for `sdpa`; unexpected results may be encountered.
=========================== short test summary info ============================
FAILED tests/ext_thunder/test_thunder_networks.py::test_hf_for_nemo[Qwen/Qwen2.5-7B-Instruct] - torch._dynamo.exc.BackendCompilerFailed: backend='<thunder.dynamo.compiler.ThunderCompiler object at 0x7fd84ecee6b0>' raised:
TypeError: tree_flatten of type <class 'torch.SymInt'> is not supported.

Set TORCH_LOGS="+dynamo" and TORCHDYNAMO_VERBOSE=1 for more information


You can suppress this exception and fall back to eager by setting:
    import torch._dynamo
    torch._dynamo.config.suppress_errors = True
====== 1 failed, 27 passed, 2 skipped, 148 warnings in 327.24s (0:05:27) =======
```

--- a/thunder/core/pytree.py
+++ b/thunder/core/pytree.py
@@ -69,6 +69,7 @@ def tree_flatten(args, namespace=OPTREE_NAMESPACE):
             torch.device,
             torch.autograd.function.FunctionCtx,
             immutable_list,
+            *torch.types.py_sym_types,
         }
         and not isinstance(args, (ProxyInterface))
         and not is_likely_from_collections_namedtuple(args)

